AudioKitSynthOne/DSP/Kernel/S1DSPKernel.mm

```diff
@@ -95,6 +97,13 @@
     }
     _rate.init();
 
+    // intialize dsp tuning table with 12ET
+    for(int i = 0; i < 128; i++) {
+        tuningTable[i].store(440. * exp2((i - 69)/12.));
+    }
+    tuningTableNPO.store(12);
```

Comment:
What's this magic number?

Reply:
Legacy default magic numbers for backwards compatibility.  Will be overwritten when the application layer tuning model initializes, which is pretty fast.

Reply:
You could do that where you declare the member variable too.. `std::atomic<int> tuningTableNPO{12};` saves a line of code :)